automatic/kitty/update.ps1

```diff
@@ -1,61 +1,31 @@
 import-module au
 
-$releases = 'https://github.com/cyd01/KiTTY/releases/latest'
-$padVersionUnder = '0.71.1'
+$latestRelease = 'https://api.github.com/repos/cyd01/KiTTY/releases/latest'
 
 function global:au_SearchReplace {
-  @{ }
-}
-
-function downloadFile($url, $filename) {
-  if ($filename.EndsWith("_nocompress.exe")) {
-    $filename = $filename -replace "_nocompress\.exe$", ".exe"
+  @{
+    ".\legal\VERIFICATION.txt" = @{
+      "(?i)(\s*zip file\:).*"           = "`${1} $($Latest.URL32)"
+      "(?i)(^\s*SHA256\:).*"            = "`${1} $($Latest.Hash)"
+      "(?i)(^\s*Get-RemoteChecksum ).*" = "`${1} $($Latest.URL32)"
+    }
   }
-
-  $filePath = "$PSScriptRoot\tools\$filename"
-  Invoke-WebRequest -UseBasicParsing -Uri $url -OutFile $filePath | Out-Null
-  return Get-FileHash $filePath -Algorithm SHA256 | % Hash
 }
 
-function global:au_BeforeUpdate {
-  $text = Get-Content -Encoding UTF8 -Path "$PSScriptRoot\VERIFICATION_Source.txt" -Raw
-  $sb = New-Object System.Text.StringBuilder($text)
-
-  $Latest.GetEnumerator() | ? { $_.Key.StartsWith("URL") } | % {
-    $fileName = $_.Key.Substring(3)
-    $checksum = downloadFile $_.Value $fileName
-    $value = [string]::Format("| {0,-20} | {1} | {2,-79} |", $fileName, $checksum, $_.Value)
-    $sb.AppendLine($value) | Out-Null
-  }
-  $value = "".PadLeft(171, '-')
-  $sb.Append("|$value|") | Out-Null
-
-  $sb.ToString() | Out-file -Encoding utf8 -FilePath "$PSScriptRoot\legal\VERIFICATION.txt"
-}
+function global:au_BeforeUpdate { Get-RemoteFiles -Purge -NoSuffix }
 
 function global:au_GetLatest {
-  $download_page = Invoke-WebRequest -UseBasicParsing -Uri $releases
-
-  [array]$uncompressedFiles = $download_page.Links | ? { $_.href -match "_nocompress\.exe" } | % href
-  $allFiles = ($download_page.Links | ? { $_.href -match "\.exe$" -and $_.href -notmatch "nocompress|portable" } | ? {
-      $name = (Split-Path -Leaf $_.href) -replace "\.exe$", "_nocompress.exe"
-      !($uncompressedFiles | ? { $_.EndsWith($name) })
-    } | % href) + $uncompressedFiles
-
-  #$allFiles = $download_page.Links | ? { $_.href -match "\.exe$" -and $_.href -NotMatch "nocompress|portable" } | % href
-
-  $version = ($allFiles | select -First 1) -split "\/v?" | select -Last 1 -Skip 1
-
-  $result = @{
-    Version = Get-FixVersion $version -OnlyFixBelowVersion $padVersionUnder
+  $assets = (Invoke-RestMethod $latestRelease).assets
+  $fileName = $assets[0].name
+  $version = $fileName.Replace("kitty-bin-", "").Replace(".zip", "")
+  $wc = [System.Net.WebClient]::new()
+  $Hash = (Get-FileHash -InputStream ($wc.OpenRead($assets[0].browser_download_url)) | Select-Object -ExpandProperty hash).ToLower()
```

Comment:
There is a AU function for this and standard method. See for example

https://github.com/majkinetor/au-packages/blob/057c7f8ede36ff4ad20b19f5c7ad91847fe1e894/rundeck/update.ps1#L22

Reply:
Done.